### PR TITLE
Changes for Windows Server unattend xml files

### PR DESCRIPTION
**Pull request by Tomorrow9**

### CODE REVIEW

File: autoinstall/Windows/win_server/bios/Autounattend.xml [hunk 1]
```diff
@@ -139,5 +145,5 @@
             </FirstLogonCommands>
         </component>
     </settings>
-    <cpi:offlineImage cpi:source="catalog://desktop-pumkr1e/users/test/downloads/share/0621/install_windows server 2019 serverdatacenter.clg" xmlns:cpi="urn:schemas-microsoft-com:cpi" />
+    <cpi:offlineImage cpi:source="wim:C:/install.wim#Windows Server 2025 SERVERDATACENTER" xmlns:cpi="urn:schemas-microsoft-com:cpi" />
```
Comment (keirazhang) [inline autoinstall/Windows/win_server/bios/Autounattend.xml:148]: Does this also work for Windows Server 2022?

Comment (Tomorrow9) [reply autoinstall/Windows/win_server/bios/Autounattend.xml:148]: Yes.

File: autoinstall/Windows/win_server/bios/Autounattend.xml [hunk 2]
```diff
@@ -30,15 +30,16 @@
...
+		    <WillShowUI>OnError</WillShowUI>
+	        </OSImage>
             </ImageInstall>
             <DiskConfiguration>
                 <Disk wcm:action="add">
                     <CreatePartitions>
                         <CreatePartition wcm:action="add">
                             <Order>1</Order>
-                            <Size>549</Size>
+                            <Size>100</Size>
```
Comment (123lzxm) [inline autoinstall/Windows/win_server/bios/Autounattend.xml:42]: Shrink the size?

Comment (Tomorrow9) [reply autoinstall/Windows/win_server/bios/Autounattend.xml:42]: Yeah, according to the size in the OS installed manually.

**Review by keirazhang — APPROVED** [review]
LGTM. Thanks!

**Review by 123lzxm — APPROVED** [review]
LGTM, thanks.

**Pull request merged**